source/vocab/things.ttl

```diff
@@ -126,6 +126,7 @@
 #TODO move to concepts.ttl?
 :genreForm a owl:ObjectProperty;
     rdfs:label "genre/form"@en, "genre/form"@sv;
+    rdfs:domain :Representation;
```

Comment:
:genreForm already has `rdfs:domain :Endeavour` (from our automatic BF2 mappings), so this is unnecessary. (In fact it's too restrictive, since using `rdfs:domain` means the owning thing *must* be of that type, and not all things with `:genreForm` are `:Representation`s.)

Reply:
True! My bad. It was not my intention to just set `:Representation` as a domain. The purpose was to "expand" the domain to include both `: Representation` and`: Endeavor` (from the bf2 declaration). But nonetheless, `:Representation` is unnecessary here.